index.d.ts

```diff
@@ -18,175 +20,29 @@ export class EventEmitter<E extends EmitSpec = [-1]> {
   /**
    * Calls each of the listeners registered.
    */
-  readonly emit: EmitMethod<E>;
+  readonly emit: (...params: Params) => boolean;
 
   /**
    * Add a listener.
    */
-  on(fn: ListenerFn<E>, context?: any): this;
-  addListener(fn: ListenerFn<E>, context?: any): this;
+  on(fn: Listener<Params>, context?: any): this;
+  addListener(fn: Listener<Params>, context?: any): this;
 
   /**
    * Add a one-time listener.
    */
-  once(fn: ListenerFn<E>, context?: any): this;
+  once(fn: Listener<Params>, context?: any): this;
 
   /**
    * Remove the listeners.
    */
-  removeListener(fn?: ListenerFn<E>, context?: any, once?: boolean): this;
-  off(fn?: ListenerFn<E>, context?: any, once?: boolean): this;
+  removeListener(fn?: Listener<Params>, context?: any, once?: boolean): this;
+  off(fn?: Listener<Params>, context?: any, once?: boolean): this;
 
   /**
    * Remove all listeners.
    */
   removeAllListeners(): this;
 }
 
-/**
- * The observable half of an event emitter.
- * Omits emit APIs so you can only bind to events, not generate them.
- */
-export type Observable<E extends EmitSpec = [-1]> = {
-  [K in 'removeListener' | 'off' | 'once' | 'on' | 'addListener']: EventEmitter<EmitSpec>[K];
-}
```

Comment:
Why was this removed?  I'm sure you have a good reason for removing it, but I haven't looked at this code in a long time so I'm confused.

Reply:
I don't know why that line was there, so I removed it. There might be some code that references it, and if there is I'll add it back, but I didn't see anywhere that is was referenced, or how someone using the package would use `Observable`.

Reply:
I think it was meant for users who want to expose only the listening API, not the emitting API, of an `EventEmitter`.  Usage might look like this:

```
interface ThingThatEmitsEvents {
  everySecond: Observable<>;
  // ....
}
function createThing(): ThingThatEmitsEvents {
  const everySecond = new EventEmitter();
  setInterval(() => onSecond.emit(), 1000);
  // 
  return {
    everySecond,
    // ...
  };
}
const thing = createThing();
thing.everySecond.on() // can register listeners
thing.everySecond.emit() // ERROR cannot emit; only gets the Observable API
```